cfssl/README.md

```diff
@@ -0,0 +1,53 @@
+# Agent Check: cfssl
+
+## Overview
+
+This check monitors [cfssl][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The cfssl check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `cfssl.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your cfssl performance data. See the [sample cfssl.d/conf.yaml][4] for all available configuration options.
```

Comment:
```suggestion
1. Edit the `cfssl.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your cfssl performance data. See the [sample cfssl.d/conf.yaml][4] for all available configuration options.
```

Reply:
Good catch − looks like the links generated by the dd scaffolding need amending. (integrations-core → integrations-extra ; and replacing the `check` by the actual check name (here `cfssl`)) − done.